### ISSUE TITLE
what is wrong

### ISSUE THREAD

**Issue by kenneth** [ontio/ontology#230]
``` 
make
go build  -ldflags "-X github.com/ontio/ontology/common/config.Version=v0.6-14-g5ee3-dirty"  -o ontology main.go
# github.com/ontio/ontology/account
account/file_store.go:34:2: undefined: keypair.ProtectedKey
make: *** [ontology] Error 2
```

**Comment by AlverLyu** [comment]
Please update ontology-crypto.

**Comment by kenneth** [comment]
钱包文件怎么生成？
No wallet.dat detected, please create a wallet by using command line.

**Comment by kenneth** [comment]
我找到了
```
./nodectl wallet --create
```

**Comment by kenneth** [comment]
跑不起来什么原因？

```
2018/05/28 15:56:23.986374 [ERROR] GID 1, Incorrectly book keepers key
panic: division by zero

goroutine 1 [running]:
math/big.nat.div(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4200be240, 0x1, 0x5, 0x0, ...)
	/usr/local/Cellar/go/1.10.2/libexec/src/math/big/nat.go:571 +0x450
```

**Comment by Arbio5zt** [comment]
1. tool nodectl has been removed.  and the ontology program can run as a CLI, you can test ontology --help and get feather informations.
2. the reason of " Incorrectly book keepers key" seems  you filled the wrong public key in config.json

**Comment by panther142** [comment]
If you are looking for a pre-configured multinode ontology privatenet, we created a docker container. See link. 

https://github.com/OntologyCommunityDevelopers/ontology-privatenet-docker

**Comment by kenneth** [comment]
是不是需要啥配置文件？

```
DefLedger.Init error genesisBlock error [Block],GenesisBlockInit err with GetBookkeeperAddress
```

**Comment by Arbio5zt** [comment]
there seems you have not clear the chain lib first.

**Comment by kenneth** [comment]
```
✗ ./ontology
2018/06/06 06:31:30.820290 [INFO ] GID 1, Config init success
Password:
Using default account:
2018/06/06 06:31:36.435821 [INFO ] GID 1, Account init success
2018/06/06 06:31:36.450810 [INFO ] GID 1, InitCurrentBlock currentBlockHash 0000000000000000000000000000000000000000000000000000000000000000 currentBlockHeight 0
2018/06/06 06:31:36.454875 [INFO ] GID 1, Ledger init success
2018/06/06 06:31:36.456275 [INFO ] GID 1, TxPool init success
2018/06/06 06:31:36.456800 [INFO ] GID 66, txpool actor started and be ready to receive txPool msg
2018/06/06 06:31:36.456919 [INFO ] GID 67, txpool-tx actor started and be ready to receive tx msg
2018/06/06 06:31:36.457011 [INFO ] GID 16, txpool-verify actor: started and be ready to receive validator's msg
2018/06/06 06:31:36.457085 [INFO ] GID 68, stateless-validator: started and be ready to receive txn
2018/06/06 06:31:36.457109 [INFO ] GID 69, stateful-validator: started and be ready to receive txn
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x14c6df7]

goroutine 1 [running]:
```

**Comment by kenneth** [comment]
git 是不是有点问题，拉代码总是冲突。
```
both modified:   README.md
```

**Comment by Arbio5zt** [comment]
could your paste the panic details and i can helps to check it ,thanks.

**Comment by kenneth** [comment]
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x14c6df7]

goroutine 1 [running]:
github.com/ontio/ontology/account.(*Account).PubKey(...)
	/GoEnv/src/github.com/ontio/ontology/account/account.go:89
github.com/ontio/ontology/p2pserver.NewServer(0x0, 0x0)
	/GoEnv/src/github.com/ontio/ontology/p2pserver/p2pserver.go:73 +0x37
main.initP2PNode(0xc4200ccc60, 0x0, 0xc4200bad10, 0x0, 0xc420080400, 0xc420063c98, 0x12766fd)
	/GoEnv/src/github.com/ontio/ontology/main.go:276 +0x57
main.startOntology(0xc4200ccc60)
	/GoEnv/src/github.com/ontio/ontology/main.go:161 +0x36c
github.com/ontio/ontology/vendor/github.com/urfave/cli.HandleAction(0x16a7440, 0x17e3ee0, 0xc4200ccc60, 0xc420080420, 0x0)
	/GoEnv/src/github.com/ontio/ontology/vendor/github.com/urfave/cli/app.go:492 +0x7c
github.com/ontio/ontology/vendor/github.com/urfave/cli.(*App).Run(0xc42009eb60, 0xc42001e1d0, 0x1, 0x1, 0x0, 0x0)
	/GoEnv/src/github.com/ontio/ontology/vendor/github.com/urfave/cli/app.go:264 +0x5ee
main.main()
	/GoEnv/src/github.com/ontio/ontology/main.go:126 +0x4f
```

**Comment by Arbio5zt** [comment]
the reason seems you have no default account in your wallet and caused panic,
we will add a default account exist check to avoid this panic, thanks for reporting 👍 .
you now can simply recreate your wallet file or check weather have ["isDefault":true] set in your wallet file.
I will attach the default wallet exist check patch to here later.

**Comment by Arbio5zt** [comment]
we have fixed in #299

**Comment by Arbio5zt** [comment]
closed for no response.
